src/java/us/kbase/mobu/tester/test/CallbackServerTest.java

```diff
@@ -510,8 +511,12 @@ public void badRelease() throws Exception {
                 "Error looking up module njs_sdk_test_1 with version " +
                  "release: 'No module version found that matches your criteria!'");
 
-        //this is the newest git commit and was registered in dev but 
-        //then the previous git commit was registered in dev
+        // this git commit was registered to dev but 
+        // then the previous git commit was registered to dev
+        // Note 7 years later - I don't understand what the above comment means.
+        // Dev versions aren't deleted if you register a new dev version AFAICT
+        // That being said, the test passes.
+        // Maybe the catalog used to replace dev versions?
```

Comment:
Yeah, I remember that it used to. There was only a single dev version at a time, and the previous was wiped. Is it not that way anymore?

Reply:
It doesn't seem to be, the test passes with dev versions that were replaced with different dev versions

Reply:
🤷